tests/test_cls.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+import unittest
+from pythainlp.cls import GzipModel
+
+
+class TestClsPackage(unittest.TestCase):
+    def test_GzipModel(self):
+        training_data =  [
+          ("รายละเอียดตามนี้เลยค่าา ^^", "Neutral"),
+          ("กลัวพวกมึงหาย อดกินบาบิก้อน", "Neutral"),
+          ("บริการแย่มากก เป็นหมอได้ไง😤", "Negative"),
+          ("ขับรถแย่มาก", "Negative"),
+          ("ดีนะครับ", "Positive"),
+          ("ลองแล้วรสนี้อร่อย... ชอบๆ", "Positive"),
+          ("ฉันรู้สึกโกรธ เวลามือถือแบตหมด", "Negative"),
+          ("เธอภูมิใจที่ได้ทำสิ่งดี ๆ และดีใจกับเด็ก ๆ", "Positive"),
+          ("นี่เป็นบทความหนึ่ง", "Neutral")
+        ]
+        model = GzipModel(training_data)
+        self.assertIsNotNone(model.predict("รู้สึกดีจัง", k=3))
```

Comment:
would it better to modify this assert to also verify the correctness of the prediction?

Reply:
I don't think we can expect much from this few-shot classifier.   May be sentiment analysis is not the best application to test  either.

Reply:
News classification seems to work well  (at least on the paper)

Reply:
> I don't think we can expect much from this few-shot classifier. May be sentiment analysis is not the best application to test either.

I generally agree, but I just wanted to make the test suite slightly less confusing.